<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ResultsSearch</class>
 <widget class="QFrame" name="ResultsSearch">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>573</width>
    <height>395</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>20</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
     <italic>false</italic>
    </font>
   </property>
   <property name="text">
    <string>Results Search</string>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>131</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Enter Details Here</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ShooterDetailsIn">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>90</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="ShooterDetailsOut">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>170</y>
     <width>256</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
